src/lib/Updates.svelte

```diff
@@ -23,18 +25,32 @@
 
   const truncatedContent = truncateContent(update.body);
 
+  const defaultImageUrl =
+    'https://zyfpmpmcpzmickajgkwp.supabase.co/storage/v1/object/public/pipeline-images/defaults/userProfile.png';
+
+    
 </script>
 
-<div class="h-[567.35px] p-9 bg-white flex-col justify-start items-start inline-flex">
-    <div class="self-stretch h-[495.35px] flex-col justify-start items-start gap-6 flex">
-        <div class="h-[188.35px] pt-[1.75px] flex-col justify-start items-start gap-[13.30px] flex">
+
+
+<div class="inline-flex flex-col items-start justify-start bg-white p-9">
+    <div class="flex flex-col items-start self-stretch justify-start gap-2">
+        <div class=" pt-[1.75px] flex-col justify-start items-start gap-[13.30px] flex">
             <div class="self-stretch text-[#282828] text-[32px] font-bold font-['Inter'] leading-10">{update.title}</div>
             <div class="self-stretch h-[62px] pb-5 border-b border-[#dcdedd] flex-col justify-start items-start gap-1 flex">
                 <div class="inline-flex items-center justify-start gap-3">
-                    <img class="w-[42px] h-[42px] relative rounded-[42px] border border-[#dcdedd]" src="https://via.placeholder.com/42x42" alt=""/>
-                    <div class="w-[120.07px] flex-col justify-start items-start inline-flex">
-                        <div class="inline-flex items-center self-stretch justify-start gap-2">
-                            <div class="w-[57.07px] h-6 text-[#282828] text-sm font-normal font-['Inter'] leading-normal">{update.userProfile.name}</div>
+                    <!-- <img class="w-[42px] h-[42px] relative rounded-[42px] border border-[#dcdedd]"  src={user.image_url && user.image_url !== '' ? user.image_url : defaultImageUrl} alt=""/> -->
+  
+      <img
+        loading="lazy"
+        src={update.userProfile.image && update.userProfile.image !== '' ? update.userProfile.image : defaultImageUrl}
```

Comment:
could we inline this into `update?.userProfile?.image !== ''` or even just `update?.userProfile?.image ? update.userProfile.image : defaultImageUrl`